eclipse/src/saros/session/resources/validation/messages.properties

```diff
@@ -0,0 +1,3 @@
+ResourceChangeValidator_0=Participants with read-only access should not edit the resources of this shared project. If you wish to makes changes, ask the host to give you write-access.
```

Comment:
You substituted the additional context (in the message names) by numbers. Don't you think it is more readable with extra context?

Reply:
The was done by the Eclipse Tooling after I got issues with the existing Message.class. Will add more meaningful names.

Reply:
Done